pipeline/api/cloud.py

```diff
@@ -57,6 +58,11 @@ def upload_file(self, file_or_path, remote_path) -> FileGet:
         else:
             return self._post_file("/v2/files/", file_or_path, remote_path)
 
+    def upload_data(self, file_or_path, remote_path) -> DataGet:
+        uploaded_file = self.upload_file(file_or_path, remote_path)
+        uploaded_data = self._post("/v2/data/", uploaded_file.dict())
```

Comment:
Could you remove the trailing slash here? Most top routes do not have one (the exception being `POST /v2/files/`, which we should fix).

Reply:
Sure, and will change on the top node too (I added a "/" to match files)